src/Game.js

```diff
@@ -95,4 +113,27 @@ export default class Game {
 
     return force;
   }
+
+  createObstacles() {
+    this.obstaclesForEasy = [new Obstacle(350, 300, 10, 70)];
+    this.obstaclesForEasy.forEach((obstacle) =>
+      this.obstacles.easy.add(obstacle)
+    );
+    this.obstaclesForMedium = [
+      new Obstacle(250, 0, 10, 400),
+      new Obstacle(450, 300, 10, 300),
+    ];
+    this.obstaclesForMedium.forEach((obstacle) =>
+      this.obstacles.medium.add(obstacle)
+    );
+    this.obstaclesForHard = [
+      new Obstacle(200, 170, 10, 200),
+      new Obstacle(0, 370, 210, 10),
+      new Obstacle(500, 340, 10, 500),
+      new Obstacle(500, 0, 10, 270),
+    ];
+    this.obstaclesForHard.forEach((obstacle) =>
+      this.obstacles.hard.add(obstacle)
+    );
```

Comment:
This could be simplified:

Modify `Group::add()` so it can accept multiple objects:
```js
add(...objects) {
  this.objects = [...this.objects, ...objects];
}
```

Then you'll be able to add objects directly into the group:
```js
this.obstacles.medium.add([
  new Obstacle(250, 0, 10, 400),
  new Obstacle(450, 300, 10, 300)
])
```